source/usage-examples/watch.txt

```diff
@@ -4,7 +4,53 @@ Watch for Changes
 
 .. default-domain:: mongodb
 
-You can keep track of changes to data in MongoDB, such as changes to a
-collection, database, or deployment, by opening a **change stream**. A change
-stream allows applications to watch for changes to data and react to them.
-The change stream returns **change event** documents when changes occur.
+You can open a change stream on a ``MongoCollection``,
+``MongoDatabase``, or ``MongoClient`` by using the ``Watch()`` method.
+
+The following example passes an aggregation pipeline to the ``Watch()``
+method, which opens a change stream on the ``haikus`` collection and
+prints inserted documents:
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/watch.go
+   :start-after: begin watch
+   :end-before: end watch
+   :emphasize-lines: 3
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/watch.go>`__ to see a fully runnable example.
+
+Expected Result
+---------------
+
+After you run the preceding example, run the :doc:`Insert a
+Document usage example </usage-examples/insertOne>` in a different
+shell. When MongoDB completes the insert operation, you should see
+output similar to the following:  
+
+.. code-block:: json
+   :copyable: false
+
+   // results truncated 
+   Waiting For Change Events. Insert something in MongoDB!
+   {fullDocument [{_id ObjectID("...")} {title Record of ...} {text No bytes, no problem....}]}
```

Comment:
```suggestion
   [{_id ObjectID("...")} {title Record of ...} {text No bytes, no problem....}]
```

If you add the `event[3].Value` from a previous comment this is how the output will look